core/ui/TestCases/DefaultTemplate.tid

```diff
@@ -72,7 +72,7 @@ code-body: yes
 		<%endif%>
 		<div class="tc-test-case-panes">
 			<div class="tc-test-case-source">
-				<$macrocall $name="tabs" tabsList="[all[tiddlers]sort[]] -[prefix<state>] -Description -Narrative -Output Output +[putfirst[]] -[has[plugin-type]]" state=<<state>> default="Output" template="$:/core/ui/testcases/DefaultTemplate/SourceTabs"/>
+				<$macrocall $name="tabs" tabsList="[all[tiddlers]sort[]] -[prefix[$:/state/]] -Description -Narrative -Output Output +[putfirst[]] -[has[plugin-type]]" state=<<state>> default="Output" template="$:/core/ui/testcases/DefaultTemplate/SourceTabs"/>
```

Comment:
Hi, @Leilei332. Good catch.

I think `-[prefix[$:/state/]]` is too aggressive as there are plenty of documentation examples where it is instructive to see the state tiddlers.  For example see https://tiddlywiki.com/#TestCases%2FRevealWidget%2FAccordionSlider. If the user is learning about how to use the reveal widget, it is useful to see the state tiddler pop into existence when the button is clicked as it gives them a clue about how things are working.

See also this discussion: https://github.com/TiddlyWiki/TiddlyWiki5/pull/7406#pullrequestreview-1629974572

Maybe there is some finer grained change which is more appropriate.

Reply:
You're right. I think using `-[prefix[$:/state/popup/export]]` is more appropriate.